src/main/resources/assets/js/app/versioning/Branch.ts

```diff
@@ -1,4 +1,4 @@
 export enum Branch {
-    DRAFT,
-    MASTER
+    DRAFT = 'draft',
+    MASTER = 'master'
```

Comment:
It started crashing quite badly for me after this change (`Branch[branch] `returns `undefined` in `UriHelper.getPortalUri()`). Whenever I select an item in the Grid there's an endless spinner in the Preview panel.